cli/cmd/stat.go

```diff
@@ -112,7 +113,10 @@ func init() {
 func requestStatsFromAPI(client pb.ApiClient, req *pb.StatSummaryRequest) (string, error) {
 	resp, err := client.StatSummary(context.Background(), req)
 	if err != nil {
-		return "", fmt.Errorf("error calling stat with request: %v", err)
+		return "", fmt.Errorf("StatSummary API error: %v", err)
+	}
+	if e := resp.GetError(); e != nil {
+		return "", fmt.Errorf("StatSummary API error: %v", e.Error)
```

Comment:
consider making these messages slightly different, to know whether we hit `err` or `resp.GetError().Error`, maybe something like `StatSummary API response error` for the second one?